### DISCUSSION TITLE
"Artifact update problem" error in lock file maintenance PR for downloading yarn 4.1.1

### DISCUSSION THREAD

**Discussion by TakaakiAndo1** [renovatebot/renovate#29991] [Request Help]
### What would you like help with?

Other

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Thanks for reading.

As the title says, I recently started getting an "Artifact update problem" error about downloading yarn 4.1.1 in the lock file maintenance PR. (I have attached a screen shot of the actual lock file maintenance PR).

As we are just using Yarn 4.1.1 in our project, we were not sure if this error is caused by a lack of configuration to the renovatebot or if it is an issue with the renovatebot itself. So we are glad you could take a look at this issue.

- Error message displayed in PR
```
! Corepack is about to download https://repo.yarnpkg.com/4.1.1/packages/yarnpkg-cli/bin/yarn.js
```

- Screenshots
![CleanShot 2024-07-02 at 22 29 47@2x](https://github.com/renovatebot/renovate/assets/18239007/0245da62-096f-4eef-b47d-fe790bd0cc5f)

### Logs (if relevant)

<details><summary>(Information-level) Logs</summary>

```

INFO: Repository started
{
  "renovateVersion": "37.421.9"
}

INFO: Dependency extraction complete
{
  "baseBranch": "master"
  "stats": {
    "managers": {
      "github-actions": {
        "fileCount": 12,
        "depCount": 64
      },
      "npm": {
        "fileCount": 7,
        "depCount": 179
      }
    },
    "total": {
      "fileCount": 19,
      "depCount": 243
    }
  }
}

INFO: PR updated (branch="renovate/ui/lock-file-maintenance")
{
  "pr": 15215
  "prTitle": "chore(deps): lock file maintenance"
}

WARN: artifactErrors (branch="renovate/ui/lock-file-maintenance")
{
  "artifactErrors": [
    {
      "lockFile": "yarn.lock",
      "stderr": "! Corepack is about to download https://repo.yarnpkg.com/4.1.1/packages/yarnpkg-cli/bin/yarn.js\n"
    }
  ]
}

INFO: Repository finished
{
  "cloned": true
  "durationMs": 65569
}

```

</details>

**Comment by TakaakiAndo1** [comment]
The issue has been resolved, so I have closed it. (There was a problem in another area.) 
Thank you!

**Reply by quipo** [reply]
out of curiosity, what was the problem?
